apps/angular/src/app/features/anime-table/anime-table.component.ts

```diff
@@ -0,0 +1,54 @@
+import { MatTableModule } from '@angular/material/table';
+import { Component, inject } from '@angular/core';
+import { AnimeApiService } from '@js-camp/angular/core/services/anime.service';
+import { Anime } from '@js-camp/core/models/anime';
+import { Observable } from 'rxjs';
+import { CommonModule, NgOptimizedImage, DatePipe } from '@angular/common';
+import { MatChipsModule } from '@angular/material/chips';
+
+import { EmptyPipe } from '../../../shared/pipes/empty.pipe';
+
+/** Anime table column names. */
+enum AnimeTableColumns {
+	Image = 'image',
+	TitleEng = 'EnglishTitle',
+	TitleJpn = 'JapaneseTitle',
+	StartDate = 'startDate',
+	Type = 'type',
+	Status = 'status',
+}
+
+/** Anime table component. */
+@Component({
+	selector: 'camp-anime-table',
+	styleUrl: './anime-table.component.css',
+	templateUrl: './anime-table.component.html',
+	standalone: true,
+	imports: [MatTableModule, CommonModule, NgOptimizedImage, MatChipsModule, EmptyPipe, DatePipe],
+})
+export class AnimeTableComponent {
+
+	/** Anime table column names. */
+	protected readonly animeColumns = AnimeTableColumns;
+
+	/** An array that determines anime table columns order. */
+	protected readonly displayedColumns = Object.values(this.animeColumns);
+
+	/** Stream containing anime data from the server. */
+	protected animeList$!: Observable<Anime[]>;
```

Comment:
You have already assigned `animeList$` value in the constructor, so you don't need non-nullable assertion anymore
```suggestion
	protected animeList$: Observable<Anime[]>;
```